src/features/embalm/stepContent/components/ArchaeologistListItem.tsx

```diff
@@ -106,7 +116,7 @@ export function ArchaeologistListItem({
         icon={false}
         checkbox={false}
       >
-        {archaeologist.profile.failures.toString()}
+        {Number(archaeologist.profile.failures).toString()}
```

Comment:
See if we can remove this `Number` casting - try running `npm install`

Reply:
Yes we can, after running npm install it stopped giving undefined